hack/update-proto.sh

```diff
@@ -18,6 +18,9 @@ HACK_DIR=$(dirname "${BASH_SOURCE}")
 
 echo "Updating plugin proto"
 
+protoc pkg/plugin/proto/backupitemaction/v1/*.proto --go_out=plugins=grpc:pkg/plugin/generated/ -I pkg/plugin/proto/
```

Comment:
I figured out how to get the output files to generate properly. Replace the above line with this:
```
protoc pkg/plugin/proto/backupitemaction/v1/*.proto --go_out=plugins=grpc:pkg/plugin/generated/ --go_opt=module=github.com/vmware-tanzu/velero/pkg/plugin/generated -I pkg/plugin/proto/
```
The required addition is `--go_opt=module=github.com/vmware-tanzu/velero/pkg/plugin/generated` which tells protoc-gen-go to strip the module prefix from the go_package because we're already under the module root dir.